<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherWindow</class>
 <widget class="QMainWindow" name="TeacherWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_11">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0" colspan="3">
       <widget class="QStackedWidget" name="stackedWidget_2">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <widget class="QWidget" name="page_1">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="2">
           <widget class="QLineEdit" name="search"/>
          </item>
          <item row="0" column="0" colspan="4">
           <widget class="QTableWidget" name="tableWidget1"/>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>导出成为表格文件</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <spacer name="horizontalSpacer_9">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="2" column="0" colspan="2">
               <widget class="QPushButton" name="admin_modify">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string>确认修改</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="admin_id"/>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>管理员账号：</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>管理员密码：</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="admin_pas"/>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_10">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <spacer name="horizontalSpacer_11">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="2" column="0">
               <widget class="QLabel" name="label_cap">
                <property name="text">
                 <string>最多容纳人数：</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEdit_teacher"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_teacher">
                <property name="text">
                 <string>老师姓名：</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QLineEdit" name="lineEdit_note"/>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="lineEdit_cap"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_sel">
                <property name="text">
                 <string>已选人数：</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="lineEdit_sel"/>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_name">
                <property name="text">
                 <string>课程名称：</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_note">
                <property name="text">
                 <string>备注：</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEdit_name"/>
              </item>
              <item row="6" column="0" colspan="2">
               <widget class="QPushButton" name="SureModify">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string>确认添加</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_typ">
                <property name="text">
                 <string>课程类型：</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLineEdit" name="lineEdit_typ"/>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_12">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_modify">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>修改授课教师为：</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>修改容纳人数为：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>需要修改的课程ID：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="modifyID"/>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="modifycap"/>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="modifyteacher"/>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QLabel" name="label_4">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>（以下内容需要修改则改动内容，无需修改保持为原样）</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QPushButton" name="modify">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>90</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>修改！</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>修改备注为：</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="modifynote"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_inquire">
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="1" column="1" colspan="2">
             <widget class="QRadioButton" name="findassistant">
              <property name="text">
               <string>查看课程报名助教名单</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="3">
             <widget class="QTableWidget" name="tableWidget_inquire"/>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="findstudent">
              <property name="text">
               <string>查看选课学生名单</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="inquire">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>查询！</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="id_inquire"/>
            </item>
            <item row="3" column="0" colspan="3">
             <widget class="QTextBrowser" name="outcome">
              <property name="font">
               <font>
                <family>Calibri Light</family>
                <pointsize>20</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="minimumSize">
               <size>
                <width>226</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>请输入课程ID：</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="idenline">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="timeline">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="Logout">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>注销登陆</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="EnterCourses">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>录入课程信息</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="InquireCourses">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>查询所有课程</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="AddCourses">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>增加课程</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="DeleteCourses">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>删除课程</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ModifyCourses">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>改变课程信息</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="CourseInformation">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>具体课程信息查询</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>个人中心</string>
    </property>
    <addaction name="actionb"/>
    <addaction name="actionc"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actiona"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actiona">
   <property name="text">
    <string>帮助文档</string>
   </property>
  </action>
  <action name="actionb">
   <property name="text">
    <string>修改管理员密码</string>
   </property>
  </action>
  <action name="actionc">
   <property name="text">
    <string>退出系统</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
